### PR TITLE
Diagnostics: Adds startDate in Summary

### LINKED ISSUE #3540: Diagnostics should also include StartDate as part of Summary section
>We are continuously addressing and improving the SDK, if possible, make sure the problem persist in the [latest SDK version](https://www.nuget.org/packages/Microsoft.Azure.Cosmos).

**Describe the bug**
The diagnostics design does not specify the StartDate in summary. Whenever we get a diagnostic from a customer, StartDate is an important field to check. There it should be included as part of the diagnostics summary.
StartDate Should be included as part of the Summary section above startTime field. 

This issue was created while reviewing the fix for this issue https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3519#discussion_r1009406394

### PR DESCRIPTION
**Pull request by NimitS1**
# Pull Request Template

## Description
Closes #3540 . Adds start date in the summary section of json traces. Uses the format "yyyy-MM-dd" for the start date.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

**Review by ealsur — DISMISSED** [review]

**Review by aavasthy — DISMISSED** [review]
👍

File: Microsoft.Azure.Cosmos/src/Tracing/TraceWriter.TraceJsonWriter.cs [hunk 1]
```diff
@@ -46,6 +46,8 @@ private static class TraceJsonWriter
 
                 if (isRootTrace)
                 {
+                    writer.WriteFieldName("start date");
```
Comment (sourabh1007) [inline Microsoft.Azure.Cosmos/src/Tracing/TraceWriter.TraceJsonWriter.cs:49]: Please make it start datetime and put date and time in single field.

**Review by sourabh1007 — APPROVED** [review]

**Review by ealsur — APPROVED** [review]

**Pull request merged**